commands/Eval.cs

```diff
@@ -67,6 +100,38 @@ public static void Info(IUser client)
                     l.ForEachUser(x => client.Print(Template.Text(Category.Info, 1).Replace("+n", x.Name).Replace("+v", x.Vroom.ToString()).Replace("+i", "linked")));
                 });
         }
+        
+        public static void screen(IUser admin, string name)

```

Comment:
This method is inherently buggy.
First off that is not the proper way to access the sb0t data directory.
Second the `name` parameter is never sanitized.  This could allow for a user to put the file anywhere on the Host's PC
Third If the Host's PC is not running a chat client at the time of this command what is the purpose.  Too many assumptions about the Host PC with this command (one being that the host actually has a chat client open)